.github/workflows/k8s-ci.yml

```diff
@@ -0,0 +1,25 @@
+name: K8s CI
+on:
+  pull_request:
+    types: ['opened', 'edited', 'reopened', 'synchronize']
+
+jobs:
+  k8s-ci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: DeterminateSystems/nix-installer-action@v11
+        with:
+          kvm: true
+      - uses: DeterminateSystems/magic-nix-cache-action@v6
+      - name: Pre-populate nix-shell
+        run: |
+          export NIX_PATH=nixpkgs=$(jq '.nixpkgs.url' nix/sources.json -r)
+          echo "NIX_PATH=$NIX_PATH" >> $GITHUB_ENV
+          nix-shell shell.nix --run "echo"
+      - name: BootStrap k8s cluster
+        run: |
+          nix-shell shell.nix --run "ls -ltr ./scripts/k8s/deployer.sh"
+          nix-shell shell.nix --run "./scripts/k8s/deployer.sh start --label"
+          nix-shell shell.nix --run "kubectl get nodes -o wide"
+          nix-shell shell.nix --run "kubectl get pods -A"
```

Comment:
I'm guessing this is just a placeholder for now to test the deployer?

Reply:
yeah, i used this for testing purpose only